app/src/main/java/edu/grinnell/appdev/grinnelldirectory/APICaller.java

```diff
@@ -0,0 +1,138 @@
+package edu.grinnell.appdev.grinnelldirectory;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.APICallerInterface;
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.DatabaseAPI;
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+import edu.grinnell.appdev.grinnelldirectory.Model.User;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+
+
+public class APICaller {
+
+    private static final String BASE_URL = "https://itwebappstest.grinnell.edu/DotNet/WebServices/api/";
+
+    private Retrofit retrofit;
+    private DatabaseAPI dbAPI;
+    private Call<List<Person>> personQuery;
+    private APICallerInterface apiCallerInterface;
+    private User user;
+
+    public APICaller(User user1, APICallerInterface apiInterface) {
```

Comment:
Please use proper variable names here. I think `user` would be enough. If you have to, for some reason, distinguish b/n the first and second user, make the variables `firstUser` and `secondUser` respectively.